MOKApp/app/src/main/java/mok/it/app/mokapp/fragments/MyBadgesFragment.kt

```diff
@@ -33,9 +38,21 @@ class MyBadgesFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        getBadges(userModel.collectedBadges)
+        projectService.getProjectsByIds(
+            projectIds = userModel.projectBadges.keys.toList(),
+            onComplete = { projectsList ->
+                collectedBadges.clear()
+                for (project in projectsList) {
+                    collectedBadges.add(Pair(project, userModel.projectBadges[project.id] ?: 0))
+                }
+                initRecyclerView()
+            },
+            onFailure = { exception ->
+                Log.d("MANCSAIM", exception.toString())
+            }
+        )
     }
-
+/*
```

Comment:
ez miért van kikommentezve?

Reply:
Átírtam, service-ből jön az adat, ez már nem kell, törlöm is